koji_wrapper/tag.py

```diff
@@ -58,12 +58,18 @@ def tagged_list(self):
     def tagged_list(self, tagged_list):
             self.__tagged_list = tagged_list
 
-    def builds(self):
+    def builds(self, **kwargs):
```

Comment:
Still trying to see how this will impact usage style having to specify just tag in __init__ and other parameters in here, some of which are available as parameters some of which are not, and if you want to re-query you would have to specify the same set of parameters in a second time instead of the class remembering the contents.  Not saying this is bad just different, and need to evaluate how it will play out.